src/minigit.hpp

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include <map>
+#include <string>
+class Minigit {
```

Comment:
Aquí faltaría añadir las instrucciones de `using`:

```suggestion
#pragma once

#include <map>
#include <string>

using std::map;
using std::string;

class Minigit {
```